docs/clients/cocoa/dsym.rst

```diff
@@ -132,7 +132,10 @@ Shell: `/bin/bash`
     export SENTRY_ORG=___ORG_NAME___
     export SENTRY_PROJECT=___PROJECT_NAME___
     export SENTRY_AUTH_TOKEN=YOUR_AUTH_TOKEN
-    sentry-cli upload-dsym
+    ERROR=$(sentry-cli upload-dsym 2>&1 >/dev/null)
+    if [ ! -z "$ERROR" ]; then
```

Comment:
hmm I guess you are right ... could improve that a bit ...
basically I am capturing the output of the command (which only returns an output if it fails) and show it to the user ...